util/cli.go

```diff
@@ -0,0 +1,12 @@
+package util
+
+import cli "github.com/urfave/cli/v2"
+
+var IsVeryVerbose bool
```

Comment:
We should just use `debug` instead of `VeryVerbose`. `debug` implies that the configuration is for debugging purposes which is what you are intending to do here.

Reply:
I agree. I think there's generally two models CLIs use: `-v/-vv/-vvv` (nmap, curl) or `--log-level` (golang things) . Maybe it makes sense to change `--logging` to `--log-level INFO` (maybe make this the default) and just add `--log-level DEBUG`, etc.

Reply:
`--log-level DEBUG/INFO` makes much more sense. Let's go with this.

Reply:
noted